lib-rca/src/main/java/com/facebook/openwifi/librca/stats/inputs/InputStats.java

```diff
@@ -0,0 +1,74 @@
+/*
+ * Copyright (c) Meta Platforms, Inc. and affiliates.
+ * All rights reserved.
+ *
+ * This source code is licensed under the BSD-style license found in the
+ * LICENSE file in the root directory of this source tree.
+ */
+
+package com.facebook.openwifi.librca.stats.inputs;
+
+/** Input Data Model */
+public class InputStats {
+    /** Radio parameters */
+    public static class Radio {
+        public long active_ms;
+        public long busy_ms;
+        public int channel;
+        public String channel_width;
+        public long noise;
+        public String phy;
+        public long receive_ms;
+        public long transmit_ms;
+        public int tx_power;
+    }
+	public static class SSID {
+		public static class Association {
+			/** Modulation and Coding Scheme */
+			public static class Rate {
+				public long bitrate;
+				public int chwidth;
+				public int mcs;
+			}
+			// bssid of the AP radio
+			public String bssid;
+			// client MAC
+			public String station;
+			public long connected;
+			public long inactive;
+			public int rssi;
+			public long rx_bytes;
+			public long rx_packets;
+			public Rate rx_rate;
+			public long tx_bytes;
+			public long tx_duration;
+			public long tx_failed;
+			public long tx_offset;
+			public long tx_packets;
+			public Rate tx_rate;
+			public long tx_retries;
+			public int ack_signal;
+			public int ack_signal_avg;
+		}
+		public Association[] associations;
+		public Radio radio;
```

Comment:
The Radio class does not need to be under SSID. Instead SSID can be under Radio.